client/src/components/create-page/CreateForm.js

```diff
@@ -73,9 +74,15 @@ export default function CreateForm({
           }}
           required
         />
-        <button className="generic-button browse-button" onClick={uploadImage}>
-          {buttonName}
-        </button>
+        {imageSelected && (
+          <button
+            className="generic-button browse-button"
+            onClick={uploadImage}
+            disabled={buttonName === "Done!" ? true : false}
+          >
+            {buttonName}
+          </button>
```

Comment:
`buttonName` being a state at a higher level in the app reads a bit more difficult than it needs to. Maybe you rather want to have a state, which holds the current "image" status and then add the respective logic here